src/app/components/NextButton.tsx

```diff
@@ -19,7 +19,7 @@ function NextButton(props) {
     nextButton = (
       <button
         onClick={selectNext}
-        className="button button--primary button--next"
+        className="button button--primary button--next button--flex"
```

Comment:
I don't think these buttons need to be primary, the blue I think should be reserved for a more prominent call to action.